### PR TITLE
Task to Bounties

### PR DESCRIPTION
**Pull request by YashovardhanSingh**
closes: #2045

### CODE REVIEW

File: src/data/bounties.json [hunk 1]
```diff
@@ -64,7 +64,7 @@
     },
     {
       "repository": "thenewboston-js",
-      "title": "[Task] Create `Bank` Documentation",
```
Comment (jamessspanggg) [inline src/data/bounties.json:67]: no need to change the content of the `.json` files

File: src/apis/roadmap/index.ts [hunk 2]
```diff
@@ -1,19 +1,19 @@
 import axios from 'axios';
 import {PaginatedResponse} from 'types/api';
-import {RoadmapTask} from 'types/roadmap';
```
Comment (jamessspanggg) [inline src/apis/roadmap/index.ts:3]: I don't think we should change the roadmap `Task` to `Bounty` as `Bounty` is offered to external contributors, while roadmap are the tasks for the core team members

**Comment by jamessspanggg** [comment]
Seems like linter is failing

**Review by thesanjeevsharma — CHANGES_REQUESTED** [review]
@YashovardhanSingh could you make the changes James mentioned.

Also, make sure you have eslint installed in vscode(or any other IDE you're using).

You can finally run `npm run lint`, to check for linting errors.

**Comment by YashovardhanSingh** [comment]
working on it

**Comment by YashovardhanSingh** [comment]
Pls, describe completely the upcoming tasks, so that there is no misinterpretation.

File: src/data/bounties.json (hunk 1 above)
Comment (thesanjeevsharma) [reply src/data/bounties.json]: done!

File: src/apis/roadmap/index.ts (hunk 2 above)
Comment (thesanjeevsharma) [reply src/apis/roadmap/index.ts]: done!

File: src/types/faq.tsx [hunk 3]
```diff
@@ -112,11 +112,11 @@ export const faqQuestionsAndAnswers: TopicQuestionAndAnswers[] = [
             incentive to solve the problems that really need solving. For example, if a crypto that is mined ever needs
             a new payment integration or mobile app, who would work on that? What would their incentive be? The idea
             that because there might be many people that own that currency, there are also going to be a lot of people
-            with the incentive to work on such tasks is not realistic. That's why, for example, numerous popular cryptos
-            have few integrations and bad looking UIs. With the incentive model of rewarding contributors an amount of
-            coins based on the value they have given to the network and/or the community, it not only allows us to
-            tackle these issues, but brings a more diverse range of contributors and community members to thenewboston
-            overall.
+            with the incentive to work on such bounties is not realistic. That's why, for example, numerous popular
```
Comment (thesanjeevsharma) [inline src/types/faq.tsx:115]: @buckyroberts not sure if we should revert this to `tasks` or `bounties` is fine?

**Review by thesanjeevsharma — APPROVED** [review]

File: src/types/faq.tsx (hunk 3 above)
Comment (buckyroberts) [reply src/types/faq.tsx:115]: Good catch, I'll revert.

**Comment by YashovardhanSingh** [comment]
is this done?

**Comment by thesanjeevsharma** [comment]
@YashovardhanSingh Yes, it got merged!

**Comment by YashovardhanSingh** [comment]
@thesanjeevsharma 853cf2c5353f1fe403672611725480dd55e6424660212924a8d60b38fdaff4a0

**Pull request merged**